django_superform/forms.py

```diff
@@ -184,7 +184,7 @@ def __getitem__(self, name):
         """
         if name not in self.fields and name in self.composite_fields:
             field = self.composite_fields[name]
-            return CompositeBoundField(self, field, name)
+            return field.get_bound_field(form, field_name)
```

Comment:
looks like `form` and `field_name` not defined here.

Should be this line substituted to `field.get_bound_field(self, name)` ?